src/node.h

```diff
@@ -305,6 +305,13 @@ NODE_EXTERN v8::Local<v8::Context> NewContext(
     v8::Local<v8::ObjectTemplate> object_template =
         v8::Local<v8::ObjectTemplate>());
 
+// Runs Node.js-specific tweaks on an already constructed context
+// Return value indicates success of operation
+NODE_EXTERN bool InitializeContext(
```

Comment:
Maybe we could also make it a `Maybe<bool>`? For example, if we want to make it part of the contract that the same context cannot run through this twice, `result.FromJust() == false` indicates that the context cannot be initialized (maybe it's already initialized), while `result.IsNothing()` indicates that a JS exception is thrown during the initialization.

Reply:
hmm, or maybe not, if we can't guarantee that we are able to always distinguish between the root cause with `result.FromJust() == false` and `result.IsNothing()` - that can be a bit difficult.